pyproject.toml

```diff
@@ -1,3 +1,5 @@
+
+ignore_missing_imports = true
```

Comment:
Can you explain this one to me?  Don't we need all our imports?

Reply:
Thanks for this catch, its suppose  to be under `[tool.mypy."pytest.*"]`
When ignore_missing_imports is set to true, mypy will not report an error when an imported module cannot be found. This is to replicate the setting that was originally found in the setup.cfg file